### DISCUSSION TITLE
Tell me about code style

### DISCUSSION THREAD

**Discussion by xpwmaosldk** [immich-app/immich#256] [General]
Hi @alextran1502 and contributors 
I am contributing to mobile.
But every time I open a pull request, I am worried that the style of the original source code will be ruined.
I would like to contribute more to this project
In order to do that, I think we have to match the code style. 
If you tell me about the rules that apply to mobile source code, I will match them.

Cheers for this project!

**Comment by alextran1502** [comment]
Here is my VScode setup for Dart

```json
  "[dart]": {
    "editor.formatOnSave": true,
    "editor.selectionHighlight": false,
    "editor.suggest.snippetsPreventQuickSuggestions": false,
    "editor.suggestSelection": "first",
    "editor.tabCompletion": "onlySnippets",
    "editor.wordBasedSuggestions": false,
    "editor.defaultFormatter": "Dart-Code.dart-code"
  },
  "dart.lineLength": 120,
```

**Comment by alextran1502** [comment]
I actually change the `lineLength` back to 80 to conform to Dart style :). So before committing, you can perform `dart format ./lib` from the `mobile` directory of the project to get the most uniform styling
